### ISSUE TITLE
Publish requirements.txt

### ISSUE THREAD

**Issue by 0xdeafbeef** [erdewit/HiFiScan#2]
Doesn't work if installed from pip.

Tried to use with these dependencies:

```
eventkit==0.8.9
numba==0.56.2
numpy==1.23.1
PyQt5==5.15.7
pyqtgraph==0.12.4
sounddevice==0.4.5
```

```
Error in task:
Traceback (most recent call last):
  File "/home/user/dev/oss/HiFiScan/hifiscan/app.py", line 351, in wrap_coro
    await coro
  File "/home/user/dev/oss/HiFiScan/hifiscan/app.py", line 56, in analyze
    async for recording in audio.record():
  File "/home/user/.local/lib/python3.10/site-packages/eventkit/event.py", line 368, in aiter
    raise args
  File "/home/user/.local/lib/python3.10/site-packages/eventkit/event.py", line 189, in emit
    result = func(obj, *args)
  File "/home/user/.local/lib/python3.10/site-packages/eventkit/ops/transform.py", line 237, in on_source
    obj = self._func(*args)
BufferError: cannot resize an array that is exporting buffers
cannot resize an array that is exporting buffers
Traceback (most recent call last):
  File "/home/user/.local/lib/python3.10/site-packages/eventkit/event.py", line 189, in emit
    result = func(obj, *args)
  File "/home/user/.local/lib/python3.10/site-packages/eventkit/ops/transform.py", line 237, in on_source
    obj = self._func(*args)
BufferError: cannot resize an array that is exporting buffers
Traceback (most recent call last):
  File "/home/user/dev/oss/HiFiScan/hifiscan/app.py", line 74, in plot
    self.plotIR()
  File "/home/user/dev/oss/HiFiScan/hifiscan/app.py", line 95, in plotIR
    t, ir = analyzer.h_inv(secs, dbRange, beta, smoothing)
AttributeError: 'NoneType' object has no attribute 'h_inv'
```

Jupiter notebook also fails to run

**Comment by unfa** [comment]
I have the same problem running via MINGW (so called GitBash) on Windows:

```
$ hifiscan
Error in task:
Traceback (most recent call last):
  File "C:\Users\unfa\AppData\Local\Programs\Python\Python310\lib\site-packages\hifiscan\app.py", line 351, in wrap_coro
    await coro
  File "C:\Users\unfa\AppData\Local\Programs\Python\Python310\lib\site-packages\hifiscan\app.py", line 56, in analyze
    async for recording in audio.record():
  File "C:\Users\unfa\AppData\Local\Programs\Python\Python310\lib\site-packages\eventkit\event.py", line 368, in aiter
    raise args
  File "C:\Users\unfa\AppData\Local\Programs\Python\Python310\lib\site-packages\eventkit\event.py", line 189, in emit
    result = func(obj, *args)
  File "C:\Users\unfa\AppData\Local\Programs\Python\Python310\lib\site-packages\eventkit\ops\transform.py", line 237, in on_source
    obj = self._func(*args)
BufferError: cannot resize an array that is exporting buffers
Traceback (most recent call last):
  File "C:\Users\unfa\AppData\Local\Programs\Python\Python310\lib\site-packages\hifiscan\app.py", line 74, in plot
    self.plotIR()
  File "C:\Users\unfa\AppData\Local\Programs\Python\Python310\lib\site-packages\hifiscan\app.py", line 95, in plotIR
    t, ir = analyzer.h_inv(secs, dbRange, beta, smoothing)
AttributeError: 'NoneType' object has no attribute 'h_inv'
```

**Comment by unfa** [comment]
After installing everything listed in this issue, I still have errors:

```
$ hifiscan
Error in task:
Traceback (most recent call last):
  File "C:\Users\unfa\AppData\Local\Programs\Python\Python310\lib\site-packages\hifiscan\app.py", line 351, in wrap_coro
    await coro
  File "C:\Users\unfa\AppData\Local\Programs\Python\Python310\lib\site-packages\hifiscan\app.py", line 56, in analyze
    async for recording in audio.record():
  File "C:\Users\unfa\AppData\Local\Programs\Python\Python310\lib\site-packages\eventkit\event.py", line 368, in aiter
    raise args
  File "C:\Users\unfa\AppData\Local\Programs\Python\Python310\lib\site-packages\eventkit\event.py", line 189, in emit
    result = func(obj, *args)
  File "C:\Users\unfa\AppData\Local\Programs\Python\Python310\lib\site-packages\eventkit\ops\transform.py", line 237, in on_source
    obj = self._func(*args)
BufferError: cannot resize an array that is exporting buffers
```

**Comment by erdewit** [comment]
Thanks for the bug report and the stack traces. This problem is fixed in v1.0.1 which is just released.

There should be no need for pinned versions of dependencies.

**Comment by exislow** [comment]
Actually you should always use pinned dependency version, since you never now, if / when breaking changes will be introduced to the dependencies. Could you please provide pinned version numbers? It would be awesome, if you would even use [Poetry](https://python-poetry.org/) for this. Thanks!
